tests/integration/conftest.py

```diff
@@ -75,3 +91,32 @@ def ensure_cleanup(name: str) -> None:
         except KeyError:
             # TODO: remove Try/Except block after CORE-767 is resolved in GX Core
             pass
+
+
+@pytest.fixture
+def load_postgres_data(
+    postgres_connection_string: str,
+    table_name: str,
+) -> Generator[Callable[[list[dict]], None], None, None]:
+    """Loads data into a table called `test_table` in the Postgres database.
+
+    This will have a string column called name, and an int column called age.
+    This should be enough to cover our use cases.
+    """
+
+    def _load_postgres_data(data: list[dict]) -> None:
+        engine = create_engine(url=postgres_connection_string)
+        with engine.connect() as conn, conn.begin():
+            conn.execute(
+                text(f"CREATE TABLE {table_name} (name VARCHAR(255), age INT);")
+            )
+            conn.execute(
+                text(f"INSERT INTO {table_name} (name, age) VALUES (:name, :age);"),
+                data,
+            )
+
+    yield _load_postgres_data
+
+    engine = create_engine(url=postgres_connection_string)
+    with engine.connect() as conn, conn.begin():
+        conn.execute(text(f"DROP TABLE {table_name};"))
```

Comment:
i think this is probably the best approach, since it's self-contained, and we don't expect to write comprehensive integration tests against all supported backends. We could also consider using our GX testing framework, for instance `PostgresBatchTestSetup`.

Reply:
Yup, considered other options, and wanted to keep it light weight until needs dictate otherwise!